frontend/src/System/Tasks/Queued/QueuedTasks.js

```diff
@@ -10,32 +10,32 @@ import QueuedTaskRowConnector from './QueuedTaskRowConnector';
 const columns = [
   {
     name: 'trigger',
-    label: '',
+    label: translate('Trigger'),
```

Comment:
This should always be blank, no translate

Reply:
squished